.yamllint.yaml

```diff
@@ -31,6 +31,8 @@ rules:
 
   line-length: disable
 
+  new-lines: disable
```

Comment:
This doesn't seem right to me. In our ESLint configuration, we enforce the default Unix-style line break. Is it not possible to do the same thing here?

Reply:
UNIX types are checked by default, so I could just revert these changes. 
On `vcbuild.bat`, though, the `lint-js` command currently takes an argument that disables checking new line types. 
Should I add, on `.gitattributes` file, a line that states that every `.yml` or `.yaml` file should have `text eol=lf`?

Thanks for the feedback!

Reply:
> Should I add, on `.gitattributes` file, a line that states that every `.yml` or `.yaml` file should have `text eol=lf`?

If that is a safe thing to do and not Windows-hostile, then yes, I think that's a good approach.

> The other option I see is to pass an argument from vcbuild.bat to the makefile (from which comes the lint command) that will make it recognize where the command started from. This way I could disable line checking for Windows users in a similar fashion to what already happens for the `lint-js` command.

I'm less clear on what exactly that would look like, but generally I think that approach would be fine too.